### ISSUE TITLE
Why is Krabby not in either Chrome or Firefox extension stores?

### ISSUE THREAD

**Issue by leira** [alexherbo2/krabby#38]
I'm so excited for Krabby as a Kakoune user. I couldn't find Krabby in either Chrome or Firefox extension stores. Krabby has been around for a while now. What is the reason it is not in the extension stores?

**Comment by David-Else** [comment]
Yes, I would love to try it, but it needs to be in the stores to add another layer of trust.
